content/en/security/logs.md

```diff
@@ -25,6 +25,25 @@ For customers using release 6, the Agent can be configured to filter Logs sent b
 
 For customers using release 6, the Agent can be configured to obfuscate specific patterns within Logs sent by the Agent to the Datadog application. To mask sensitive sequences within your Logs, use the `log_processing_rules` [setting][4], with the  **mask_sequences** `type`. This setting enables the creation of a list containing one or more regular expressions, which instructs the Agent to redact sensitive data within your Logs.
 
+## Configuration Requirements for HIPAA-enabled Customers
+
+Datadog will sign a Business Associate Agreement (BAA) with customers that transmit protected health information (ePHI) via Datadog’s Log Management Service.
+
+Prior to executing a BAA, customers transmitting ePHI to the Datadog Log Management Service must implement the following configurations:
+
+* The Datadog agent must be configured to submit logs to `encrypted-intake.logs.datadoghq.com`
```

Comment:
It will surely be tcp-encrypted-intake.logs.datadoghq.com

Reply:
Confirmed: tcp-encrypted-intake.logs.datadoghq.com

Reply:
Thanks @gaetan-deputier I've updated the doc accordingly.